vault/core.go

```diff
@@ -1129,18 +1133,40 @@ func (c *Core) sealInternalWithOptions(grabStateLock, keepHALock bool) error {
 	c.clearForwardingClients()
 	c.requestForwardingConnectionLock.Unlock()
 
+	activeCtxCancel := c.activeContextCancelFunc.Load().(context.CancelFunc)
```

Comment:
You should nil check this (after asserting; see https://play.golang.org/p/x7ohO9AbkID)

Reply:
oh nm I see you did it below